sysboundary/ionosphere.cpp

```diff
@@ -1102,6 +1102,112 @@ namespace SBC {
 
    }//modifiedMidpoint Method
 
+   void SphericalTriGrid::dormandPrinceStep( 
+      std::array<Real, 3>& r,
+      std::array<Real, 3>& b,
+      Real& step,
+      Real maxStepsize,
+      TracingFieldFunction& BFieldFunction,
+      bool outwards
+    ){
+
+      Real k1x,k2x,k3x,k4x,k5x,k6x,k7x;
+      Real k1y,k2y,k3y,k4y,k5y,k6y,k7y;
+      Real k1z,k2z,k3z,k4z,k5z,k6z,k7z;
+      
+      assert(r.size()==3 && "Erroneus size for R");
```

Comment:
Well, r is an ```array<Real, 3>```, so this assert is *always true*

Reply:
Indeed. I actually lifted this off of my mini app which uses vectors so these stayed there. Doing all those changes in a minute